docs/en/connector-v2/sink/SqlServer.md

```diff
@@ -0,0 +1,171 @@
+    user = SA
+    password = "Y.sa123456"
+    query = "select * from column_type_test.dbo.full_types_jdbc"
+    # Parallel sharding reads fields
+    partition_column = "id"
+    # Number of fragments
+    partition_num = 10
+
+  }
+  # If you would like to get more information about how to configure seatunnel and see full list of source plugins,
+  # please go to https://seatunnel.apache.org/docs/connector-v2/source/Jdbc
+}
+
+transform {
+
+  # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+  # please go to https://seatunnel.apache.org/docs/transform-v2/sql
+}
+
+sink {
+  Jdbc {
+    driver = com.microsoft.sqlserver.jdbc.SQLServerDriver
+    url = "jdbc:sqlserver://localhost:1433;databaseName=column_type_test"
+    user = SA
+    password = "Y.sa123456"
+    query = "insert into full_types_jdbc_sink( id, val_char, val_varchar, val_text, val_nchar, val_nvarchar, val_ntext, val_decimal, val_numeric, val_float, val_real, val_smallmoney, val_money, val_bit, val_tinyint, val_smallint, val_int, val_bigint, val_date, val_time, val_datetime2, val_datetime, val_smalldatetime ) values( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )"
+
+  }  # If you would like to get more information about how to configure seatunnel and see full list of sink plugins,
+  # please go to https://seatunnel.apache.org/docs/connector-v2/sink/Jdbc
+}
+```
+
+### CDC(Change data capture) event
+
+> CDC change data is also supported by us In this case, you need config database, table and primary_keys.
+
+```
+Jdbc {
+  source_table_name = "customers"
+  driver = com.microsoft.sqlserver.jdbc.SQLServerDriver
+  url = "jdbc:sqlserver://localhost:1433;databaseName=column_type_test"
+  user = SA
+  password = "Y.sa123456"
+  generate_sink_sql = true
+  database = "column_type_test"
+  table = "dbo.full_types_sink"
+  batch_size = 100
+  primary_keys = ["id"]
+}
+```
+
+### Exactly Once Sink
+
+> Transactional writes may be slower but more accurate to the data
+
+```
+  Jdbc {
+    driver = com.microsoft.sqlserver.jdbc.SQLServerDriver
+    url = "jdbc:sqlserver://localhost:1433;databaseName=column_type_test"
+    user = SA
+    password = "Y.sa123456"
+    query = "insert into full_types_jdbc_sink( id, val_char, val_varchar, val_text, val_nchar, val_nvarchar, val_ntext, val_decimal, val_numeric, val_float, val_real, val_smallmoney, val_money, val_bit, val_tinyint, val_smallint, val_int, val_bigint, val_date, val_time, val_datetime2, val_datetime, val_smalldatetime ) values( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? )"
+    is_exactly_once = "true"
+
+    xa_data_source_class_name = "com.microsoft.sqlserver.jdbc.SQLServerXADataSource"
+
+  }  # If you would like to get more information about how to configure seatunnel and see full list of sink plugins,
+  # please go to https://seatunnel.apache.org/docs/connector-v2/sink/Jdbc
+
```

Comment:
PTAL: @ic4y @hailin0 I don't know whether the configuration is correct here. Please help me review thks